iot_oca/tests/test_iot.py

```diff
@@ -1,6 +1,13 @@
 # Copyright (C) 2018 Creu Blanca
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-from mock import patch
+
+try:
+    from mock import patch
+except ImportError:
+    import os
+
+    os.system("pip install mock")
```

Comment:
Don't do that

Keep as 

`from mock import patch`

If you don't have this installed, the only problem will be a failure in test, and that is not important.